internal/bminventory/inventory.go

```diff
@@ -1062,15 +1058,12 @@ func (b *bareMetalInventory) GenerateClusterISOInternal(ctx context.Context, par
 		return nil, common.NewApiError(http.StatusInternalServerError, errors.New(msg))
 	}
 
-	staticIpsConfig := formatStaticIPs(params.ImageCreateParams.StaticIpsConfig)
-
-	vlansConfig := formatVlans(params.ImageCreateParams.VlansConfig)
+	staticNetworkConfig := formatStaticNetwork(params.ImageCreateParams.StaticNetworkConfig)
 
 	var imageExists bool
 	if cluster.ImageInfo.SSHPublicKey == params.ImageCreateParams.SSHPublicKey &&
 		cluster.ProxyHash == clusterProxyHash &&
-		cluster.ImageInfo.StaticIpsConfig == staticIpsConfig &&
-		cluster.ImageInfo.VlansConfig == vlansConfig &&
+		cluster.ImageInfo.StaticNetworkConfig == staticNetworkConfig &&
```

Comment:
Isn't it breaking existing API?

Reply:
it does. we are deleting the current API, and the PR for disabling the e2e and test-infra test are already pushed